builtin/mainmenu/tab_multiplayer.lua

```diff
@@ -33,8 +40,10 @@ local function get_formspec(tabview, name, tabdata)
 		"field[8,1.95;2.95,0.5;te_name;;" ..
 			core.formspec_escape(core.setting_get("name")) .. "]" ..
 		"pwdfield[10.78,1.95;1.77,0.5;te_pwd;]" ..
-		"box[7.73,2.35;4.3,2.28;#999999]"
-
+		"box[7.73,2.35;4.3,2.28;#999999]"..
+		"field[0.15,0.25;5,0.27;te_search;;"..core.formspec_escape(tabdata.search_for).."]"..
+		"button[4.8,0;3,0.1;btn_mp_search;" .. fgettext("Search") .. "]"
```

Comment:
the `creative` mod has uses two buttons like: `[?] [X]`
Not sure if it's worth being consistent